ubuntu24.04/precompiled/Dockerfile

```diff
@@ -55,12 +55,11 @@ ADD local-repo.sh /tmp
 RUN mkdir -p /usr/local/repos && \
     /tmp/local-repo.sh download_driver_package_deps && \
     /tmp/local-repo.sh build_local_apt_repo && \
+    # Remove the ubuntu public ubuntu apt repo to ensure we only pull from the local apt repo
+    rm /etc/apt/sources.list.d/ubuntu.sources && \
```

Comment:
Wouldn't it be better to remove all filed under `/etc/apt/sources.list.d` since we configure the local repo in the top level `/etc/apt/sources.list` file?

Reply:
Yeah, I can do that. Right now, ubuntu.sources is the only file under that directory. But, that could change in the future